tests/test_unique_queue.py

```diff
@@ -125,3 +125,17 @@ def test_should_create_unique_worker_with_ttl(self):
         expect(
             self.redis.ttl(f"resque:worker:{str(worker)}:started")
         ).Not.to_equal(-1)
+
+    def test_should_prune_dead_members_on_startup(self):
+        config.prune_dead_members = True
+        worker = UniqueWorker(server=self.redis, queues=["Detect"])
+
+        self.redis.sadd("resque:workers", "teste1")
+        members = self.redis.smembers("resque:workers")
+        expect(len(members)).to_equal(2)
+
+        worker.startup()
+
+        members = self.redis.smembers("resque:workers")
+        expect(len(members)).to_equal(1)
+        expect(str(worker).encode() in members).to_be_true()
```

Comment:
Aren't we missing a `test_should_not_prune_dead_members_on_startup`. I think it's important to check both conditions.

Reply:
@scorphus Makes sense, what do you think of the test?